docs/source/examples/zne_logical_rb_cirq_stim.md

```diff
@@ -0,0 +1,303 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3.10.4 64-bit ('folded-stim-ds-zne2')
+  language: python
+  name: python3
+---
+
+# ZNE on Stim backend with Cirq: Logical randomized benchmarking circuits
+
++++
+
+This tutorial demonstrates a method of combining quantum error mitigation (QEM) and quantum error correction (QEC), with the goal of reducing the effective logical error rate of the computation. 
+While QEM techniques such as zero noise extrapolation (ZNE) and probabilistic error cancellation are typically thought of as belonging to the NISQ regime, recently it has been shown that they can also benefit applications within the fault-tolerant regime {cite}`Piveteau_2021_PRL, Suzuki_2022_PRX, Wahl_2023_arXiv_ds_zne`.
+In this example, we will apply ZNE with noise scaling by [global unitary folding](../guide/zne-3-options.md#unitary-folding) on logical randomized benchmarking (RB) circuits.
+This tutorial also introduces the use of Mitiq's ZNE functions with Cirq as the frontend and the [Stim](https://github.com/quantumlib/Stim) stabilizer simulator as the backend.
+
++++
+
+## Setup
+
++++
+
+Since Stim is not a Mitiq-supported frontend, and the [stimcirq](https://pypi.org/project/stimcirq/) library has a conversion utility for Cirq and Stim circuits, we will use Cirq as the frontend for this example.
+
+```{code-cell} ipython3
+import cirq
+import matplotlib as mpl
+import numpy as np
+import stim
+import stimcirq
+import mitiq
+```
+
+In this example, we assume a physical error rate of 0.004 and a threshold of 0.009.
+Since the physical error rate is below the threshold, the computation is operating in the fault-tolerant regime.
```

Comment:
Question just for my understanding, and doesn't need to go in the tutorial:

If the physical error rate is above the threshold can you still combine QEM with QEC to get something useful?

Reply:
Generally not a good idea to implement QEC if p_err > p_th because the encoding will, if anything, worsen the errors instead of correct them. We don't have QEM techniques that would change that, other than noise suppression techniques, but in that case one could maybe argue that p_err could be taken as the noise-suppressed error rate.